README.md

```diff
@@ -1 +1,12 @@
 # Soccer-Lightweight-2025
+
+## Serial Communication in IRRing
+
+- `Serial1` is used for communication because we work with a **Teensy 4.1**.  
+- This should be changed depending on the microcontroller being used.  
+- The baud rate is set to `115200`.  
+- Ensure no other device uses `Serial1` simultaneously to avoid conflicts.
```

Comment:
maybe for later but can we have a simple mapping for serial and thing that points

SerialX -> used for ...

maybe it would be useful